### PR TITLE
Development/weston integration

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

**Comment by HaseenaSainul** [comment]
@mhughesacn, xdg-shell-client-protocol.c and xdg-shell-client-protocol.h are Wayland provided files and we kept copy inside compositor. There is no change by metro on this file, hence no need to update copyright of this files. Even-though the Blackduck failing on the license check on this file. Can we ignore this ?

**Comment by mhughesacn** [comment]
Hi Haseena: The BlackDuck complaint is about two other files - I think I approved the xdg files sometime earlier.  I upvoted this because RDK have a pull request active to fix the two files here (and other stuff).

**Comment by HaseenaSainul** [comment]
@mhughesacn, ok let me check

**Comment by HaseenaSainul** [comment]
@mhughesacn, is this possible to get the report file: https://protex.cmf.code.rdkcentral.com/p/projects/c_cmfrdkcentralthunderclientlibraries_7586 ?

**Comment by mhughesacn** [comment]
Sorry Haseena - you should not need the log file now.  I cleared off the violation yesterday because the attribution I wanted is coming in pull/111.  (Blackduck reports can be very hard to follow anyway.)

**Comment by HaseenaSainul** [comment]
Thanks @mhughesacn. So hope no action is required from side. And will wait for your update to proceed this PR

**Comment by mhughesacn** [comment]
Hi @HaseenaSainul : I have no further input (nothing open for compliance).  I presume you will need component owner approval.
Thanks, Martin

**Comment by HaseenaSainul** [comment]
thanks @mhughesacn.  Will ask @pwielders for approval

**Review by MFransen69 — APPROVED** [review]
